### ISSUE TITLE
_debounceMouseMoveHandler is not a function

### ISSUE THREAD

**Issue by xavier78** [avocode/avocode#1621]
* **Version**: 2.15.2

* **Web/Desktop**: Desktop

* **Platform**: darwin

* **Avocode UID**: 39965
---

```
TypeError: t._debounceMouseMoveHandler is not a function
    at t.<anonymous> (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:2436619)
    at emitOne (events.js:101:20)
    at t.emit (events.js:191:7)
    at r._handleMouseMove (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:3056970)
    at Object.invokeGuardedCallback (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-dom/lib/ReactErrorUtils.js:26:5)
    at executeDispatch (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-dom/lib/EventPluginUtils.js:85:21)
    at Object.executeDispatchesInOrder (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-dom/lib/EventPluginUtils.js:108:5)
    at executeDispatchesAndRelease (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-dom/lib/EventPluginHub.js:43:22)
    at executeDispatchesAndReleaseTopLevel (/Applications/Avocode.app/Contents/
```

**Comment by jakubzitny** [comment]
Hi @xavier78, thanks for reporting this. When exactly is it happening, please? We already know about this bug and we'll be trying to fix it as soon as possible. Please give us any more info about it if you have it 😛

**Comment by jakubzitny** [comment]
Hi, we've just released Avocode `2.14.4` where this issue has been resolved. Please update the app and let us know if you have any more problems 😉
